java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -197,9 +194,9 @@ public static TimestampedEntry parseTimestampedEntry(InputStream in) {
    * @throws SerializationException if an I/O error occurs.
    * @return {@link Ct.X509ChainEntry} proto object.
    */
-  public static Ct.X509ChainEntry parseX509ChainEntry(InputStream in, ByteString x509Cert) {
+  public static Ct.X509ChainEntry parseX509ChainEntry(InputStream in, byte[] x509Cert) {
```

Comment:
It's only called internally. I'm not sure why it's marked `public`?

But this whole series of PRs is changing the API, as the data structures are what end-users get from this...